packages/amplication-client/src/util/BillingFeature.ts

```diff
@@ -7,4 +7,5 @@ export enum BillingFeature {
   CodePushToGit = "feature-code-push-to-git",
   IgnoreValidationCodeGeneration = "feature-ignore-validation-code-generation",
   HideNotifications = "feature-ignore-validation-code-generation-hide-notifications",
+  BitbucketProvider = "feature-bitbucket",
```

Comment:
We have a kind of convention here
```suggestion
  Bitbucket = "feature-bitbucket",
```